http/server.go

```diff
@@ -561,7 +561,8 @@ func (h *DrandHandler) getBeaconHandler(chainHash []byte) (*beaconHandler, error
 	bh, exists := h.beacons[chainHashStr]
 
 	if !exists {
-		return nil, fmt.Errorf("there's no BeaconHandler for beaconHash %s", chainHash)
+		return nil, fmt.Errorf("there is no BeaconHandler for beaconHash [%s]. "+
```

Comment:
Print it in hexadecimal since these are just bytes right now and it doesn't render well

Reply:
We are using **chainHashStr** instead of **chainHash**, so it is not hex. It is a string.

Reply:
Oh right sorry all good.

Reply:
Great! I have to checked it to be sure haha. Thanks!